.github/workflows/docker-jupyter-image.yml

```diff
@@ -0,0 +1,60 @@
+name: ghcr-jupyter
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
Do we want to be more selective and only push on tags?

Reply:
I think yes in the long run but not at the moment... in general we shall do that only if it becomes and issue